gems/pending/MiqVm/MiqVm.rb

```diff
@@ -48,6 +48,12 @@ def initialize(vmCfg, ost = nil)
       @vmConfig = VmConfig.new(getCfg(@ost.snapId))
       $log.debug "MiqVm::initialize: @vmConfig.getHash = #{@vmConfig.getHash.inspect}"
       $log.debug "MiqVm::initialize: @vmConfig.getDiskFileHash = #{@vmConfig.getDiskFileHash.inspect}"
+    elsif @ost.miq_hyperv
```

Comment:
Might as well add another `# TODO: move this to` comment before this `elsif` as well.